### PR TITLE
2196 - Ungate Disability Combined Rating Calculator e2e-test.

### PR DESCRIPTION
**Pull request by tlei123**
## Description
[2196](https://github.com/department-of-veterans-affairs/va.gov-team/issues/2196) - Ungate e2e test for Disability Combined Rating Calculator.

This test was not ungated when the feature launched to Prod.

## Testing done
Cannot test locally; the gating hid this test on Prod environment.
This should be checked after deployment to Prod.

### CODE REVIEW

File: src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js [hunk 1]
```diff
@@ -29,8 +28,6 @@ module.exports = E2eHelpers.createE2eTest(client => {
     .waitForElementPresent(componentSelector, Timeouts.slow);
 
   // Test row deletion.
-  // TODO: Once Design Team has spec'ed out the relevant UX,
-  // cover edge-case for deleting row AFTER calculating combined-rating.
```
Comment (tlei123) [inline src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js]: Deleting this TODO comment - UX has been alerted to this edge-case, but has not committed to addressing it.

Comment (ncksllvn) [reply src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js]: Looks like we just need to change the `drcPagePath` value to be `'/disability/about-disability-ratings/` and I think we're good to go!

Comment (tlei123) [reply src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js]: Just fixed that & pushed it up.  :)

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by rianfowler — APPROVED** [review]
